lfs/hook.go

```diff
@@ -22,6 +27,15 @@ type Hook struct {
 	Upgradeables []string
 }
 
+// NewStandardHook creates a new hook using the template script calling 'git lfs theType'
+func NewStandardHook(theType string, upgradeables []string) *Hook {
+	return &Hook{
+		Type:         theType,
+		Contents:     strings.Replace(hookBaseContent, "__CMD__", theType, -1),
```

Comment:
That would be a lot of cruft for what will boil down to a global find/replace even with a template.  If you like I can make the string look like a template and still use `strings.Replace` but I see no reason to clutter this with all the init/struct param/execute patterns of templates.